### PR TITLE
Detailing audio file requirements

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8233

source of information https://docs.microsoft.com/en-us/powershell/module/skype/import-csonlineaudiofile?view=skype-ps

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/routing-calls-to-unassigned-numbers.md [hunk 1]
```diff
@@ -66,6 +66,8 @@ New-CsTeamsUnassignedNumberTreatment -Identity TR1 -Pattern "^\+1555333\d{4}$" -
 
 - To route calls to unassigned Microsoft Calling Plan service numbers, your tenant needs to have at least one Phone System – Virtual User license.
 
+- The custom audio file supported formats are WAV (uncompressed, linear PCM with 8/16/32-bit depth in mono or stereo), WMA (mono only), and MP3. The audio file content cannot be more 5MB.
```
Comment (JohanFreelancer9) [inline Teams/routing-calls-to-unassigned-numbers.md:69]:
```suggestion
- The custom audio file supported formats are WAV (uncompressed, linear PCM with 8/16/32-bit depth in mono or stereo), WMA (mono only), and MP3. The audio file content cannot be more than 5 MB.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
@CarolynRowe Hello Carolyn, can this be merged? Thank you

**Comment by get-itips** [comment]
@tiburd Can you help me to get this merged? Thank you

**Pull request merged**